extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
+## [UNRELEASED]
+
 ## 1.6.3 - 4 April 2022
 
 No user facing changes.
```

Comment:
Perhaps we could simply remove the ## 1.6.2 header below in this PR for simplicity?

Reply:
Sure. That is fine.